api/utils/utils.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/SAP/sap-btp-service-operator/api"
+	"github.com/go-logr/logr"
+)
+
+func ValidateNonTransientTimestampAnnotation(log logr.Logger, resource api.SAPBTPResource) error {
+
+	sinceAnnotation, exist, err := GetTimeSinceIgnoreNonTransientAnnotationTimestamp(log, resource)
+	if err != nil {
+		return err
+	}
+	if exist && sinceAnnotation < 0 {
+		return fmt.Errorf("annotation %s cannot be a future timestamp", api.IgnoreNonTransientErrorTimestampAnnotation)
```

Comment:
@danielforsap review error message

Reply:
@I065450  Pls give me a concrete example.
I am trying to understand the context.
Also, what comes before and after these messages? Are they wrapped in larger text or standalone sentences?
Thanks

Reply:
@I065450 Annotation cannot be a date in the future.